pkg/config/parameters.go

```diff
@@ -183,6 +184,7 @@ func (n NamespacedName) Validate() error {
 // TLSParameters holds configuration file TLS configuration details.
 type TLSParameters struct {
 	MinimumProtocolVersion string `yaml:"minimum-protocol-version"`
+	MaximumProtocolVersion string `yaml:"maximum-protocol-version"`
```

Comment:
Please document this in https://projectcontour.io/docs/main/configuration/#tls-configuration